Day8-Assignment/src/App.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import HeaderComponent from './components/HeaderComponent.js'
+import BodyComponent from './components/BodyComponent.js'
+import { createBrowserRouter, RouterProvider, Outlet } from "react-router-dom"
+import ErrorComponent from './components/ErrorComponent.js'
+import AboutUs from './components/AboutUs.js'
+import MemberData from './components/MemberData.js'
+import ProfileComponent from './Components/ProfileComponent.js'
+
+const App = () => {
+
+    return (
+        <>
+            <HeaderComponent />
+            <Outlet />
+        </>
+    )
+}
+
+const appRouter = createBrowserRouter([
+    {
+        path: "/",
+        element: <App />,
+        errorElement: <ErrorComponent />,
+        children: [
+            {
+                path: "/member/:username",
+                element: <MemberData />,
+
+            },
+            {
+                path: "/search",
+                element: <BodyComponent />,
+
+            },
+            {
+                path: "/about-us",
+                element: <AboutUs />,
+                children:[
+                    {
+                        path:"profile",
+                        element:<ProfileComponent  name="Rasika from NRB"/>
+                    }
```

Comment:
Can you make the component  dynamic?

Reply:
> Great work ! Added comments. Would love to see the deployed app.

thanks Pujarini , will implement all the review comments